apps/potlock/widget/Pots/Detail.jsx

```diff
@@ -171,7 +171,44 @@ props.navOptions = [
   },
 ];
 
-if (!props.nav) props.nav = "projects"; // default to home tab
+const potDetail = state.potDetail;
+const now = Date.now();
+const applicationNotStarted = now < potDetail.application_start_ms;
+const applicationOpen = now >= potDetail.application_start_ms && now < potDetail.application_end_ms;
+const publicRoundNotStarted = now < potDetail.public_round_start_ms;
+const publicRoundOpen =
+  now >= potDetail.public_round_start_ms && now < potDetail.public_round_end_ms;
+const publicRoundClosed = now >= potDetail.public_round_end_ms;
+
+const userIsAdminOrGreater =
+  potDetail.admins.includes(context.accountId) || ownerId === context.accountId;
+const userIsChefOrGreater = userIsAdminOrGreater || potDetail.chef === context.accountId;
+
+const canPayoutsBeSet = publicRoundClosed && userIsChefOrGreater && !potDetail.cooldown_end_ms;
```

Comment:
This will only work for Chefs or admins because of the `userIsChefOrGreater` check. 

change to `payoutsPending`

`const payoutsPending = publicRoundClosed && !potDetail.cooldown_end_ms;`